test/DynamoCoreTests/DynamoDefects.cs

```diff
@@ -114,7 +114,7 @@ public void Defect_MAGN_1206()
             RunModel(openPath);
 
             var add = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace<DSFunction>("ccb2eda9-0966-4ab8-a186-0d5f844559c1");
-            Assert.AreEqual(20, add.GetCachedValueFromEngine(CurrentDynamoModel.EngineController).Data);
+            AssertPreviewValue(add.GUID.ToString(), 20);
```

Comment:
From a `Guid` string to `NodeModel`, just to get its `Guid` string? Maybe we no longer needs `NodeFromWorkspace` ;)